engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -6,7 +6,9 @@ import cromwell.backend.{BackendInitializationData, BackendJobDescriptor, Backen
 import cromwell.core.logging.WorkflowLogging
 import cromwell.engine.workflow.lifecycle.execution.EngineJobExecutionActor._
 import cromwell.engine.workflow.lifecycle.execution.JobPreparationActor.{BackendJobPreparationFailed, BackendJobPreparationSucceeded}
-import cromwell.jobstore._
+import cromwell.jobstore.JobStoreService.{JobComplete, JobNotComplete, QueryJobCompletion}
+import cromwell.jobstore.{Pending => _, _}
```

Comment:
I guess you could think of it as anonymizing.  It works out to "don't import this one thing", as it collides with another `Pending` type in this file and isn't used anyways.